src/notation/view/widgets/editstyle.cpp

```diff
@@ -2360,42 +2365,22 @@ void EditStyle::setValues()
     spinFBLineHeight->setValue(styleValue(StyleId::figuredBassLineHeight).toDouble() * 100.0);
 
     QString mfont(styleValue(StyleId::musicalSymbolFont).value<String>());
+    QString dynFont(styleValue(StyleId::dynamicsFont).value<String>());
+    QString tfont(styleValue(StyleId::musicalTextFont).value<String>());
     int idx = 0;
     for (const auto& i : engravingFonts()->fonts()) {
         if (QString::fromStdString(i->name()).toLower() == mfont.toLower()) {
             musicalSymbolFont->setCurrentIndex(idx);
-            break;
         }
-        ++idx;
-    }
-
-    QString dynFont(styleValue(StyleId::dynamicsFont).value<String>());
-    idx = 0;
-    for (const auto& i : engravingFonts()->fonts()) {
         if (QString::fromStdString(i->name()).toLower() == dynFont.toLower()) {
             dynamicsFont->setCurrentIndex(idx);
-            break;
+        }
+        if ((QString::fromStdString(i->family()) + " Text").toLower() == tfont.toLower()) {
+            musicalTextFont->setCurrentIndex(idx);
         }
         ++idx;
     }
 
-    musicalTextFont->blockSignals(true);
-    musicalTextFont->clear();
-    // CAUTION: the second element, the itemdata, is a font family name!
-    // It's also stored in score file as the musicalTextFont
-    musicalTextFont->addItem("Leland Text", "Leland Text");
-    musicalTextFont->addItem("Bravura Text", "Bravura Text");
-    musicalTextFont->addItem("Emmentaler Text", "MScore Text");
```

Comment:
Note that this is the odd one out

Reply:
This behaviour is kept by using fontFamilyName in line 832 of the same file.

Reply:
But here the Display name and the Family name is different

Reply:
Yes, that is why line 832 is `musicalTextFont->addItem(fontDisplayName + " Text", fontFamilyName + " Text");`.
For Emmentaler / MScore, `fontDisplayName == "Emmentaler"` and `fontFamilyName == "MScore"`, so the behaviour is preserved (same for Gonville / Gootville).